flocker/node/agents/functional/test_gce.py

```diff
@@ -144,6 +141,25 @@ def test_multiple_cluster(self):
                          list(x.dataset_id
                               for x in gce_block_device_api_2.list_volumes()))
 
+    def test_create_duplicate_dataset_ids(self):
+        """
+        Two :class:`GCEBlockDeviceAPI` instances can be run with different
+        cluster_ids. Volumes in one cluster do not show up in listing from the
+        other.
```

Comment:
This test docstring doesn't seem to match the code in the test. Copy paste error?